packages/cli/package.json

```diff
@@ -57,7 +55,7 @@
     "@inquirer/confirm": "^2.0.15",
     "@inquirer/input": "^1.2.14",
     "@inquirer/select": "^1.3.1",
-    "@mermaidchart/sdk": "workspace:^",
+    "@mermaidchart/sdk": "^0.2.1-alpha.0",
```

Comment:
Do we need to change this? Pnpm release will automatically put the correct version right?

Reply:
> Do we need to change this? Pnpm release will automatically put the correct version right?

Yep, but `npm publish` won't. And the `workspace^` breaks with things like https://gitpkg.vercel.app, which makes testing things in git subfolders more difficult.